src/polyswarmd/__init__.py

```diff
@@ -28,7 +28,10 @@
 
 # Set up our app object
 app = Flask(__name__)
-app.config['POLYSWARMD'] = Config.auto()
+_config = Config.auto()
+app.config['POLYSWARMD'] = _config
+# Setting this value works even when Content-Length is omitted, we must have it
+app.config['MAX_CONTENT_LENGTH'] = _config.max_artifact_size * _config.artifact_limit
```

Comment:
Is this still needed?

Reply:
I think it is good to have in case someone sends us way oversize files. I'd rather have Flask/Werkzeug handle that, rather than doing our check. 

Though this does create problems since there is still some extra information regarding boundary and such in a multi part upload that would prevent all full size files.

Reply:
And it puts us back into the habbit of specifying that value in multiple places.